packages/sdk/ts/tally/results.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  MACI__factory as MACIFactory,
+  Tally__factory as TallyFactory,
+  Poll__factory as PollFactory,
+} from "@maci-protocol/contracts/typechain-types";
+
+import type { IGetResultPerOptionArgs, IGetResultsArgs } from "./types";
+
+/**
+ * Get result per option
+ * @param param0 - The get result per option args
+ * @returns The result per option
```

Comment:
We could also specify the return data (this works for automated documentation tools right?):

```javascript
/*
* @returns {bigint} - The result per option
*/
```

Reply:
it should be inferred automatically by typedoc